src/main/java/com/impactupgrade/nucleus/service/segment/MailchimpCommunicationService.java

```diff
@@ -421,6 +424,12 @@ protected MemberInfo toMcMemberInfo(CrmContact crmContact, Map<String, Object> c
     mcContact.merge_fields.mapping.put(FIRST_NAME, crmContact.firstName);
     mcContact.merge_fields.mapping.put(LAST_NAME, crmContact.lastName);
     mcContact.merge_fields.mapping.put(PHONE_NUMBER, crmContact.mobilePhone);
+
+    mcContact.consents_to_one_to_one_messaging = true; //?
```

Comment:
@VSydor Do the docs mention what this field is, how it differs from `sms_subscription_status`, and if this is required? No concerns, just curious.